src/super_gradients/training/pipelines/pipelines.py

```diff
@@ -356,14 +356,14 @@ def _decode_model_output(self, model_output: Union[List, Tuple, torch.Tensor], m
         :param model_input:     Model input (i.e. images after preprocessing).
         :return:                Predicted Bboxes.
         """
-        all_poses, all_scores = self.post_prediction_callback(model_output)
-
+        result = self.post_prediction_callback(model_output)
         predictions = []
-        for poses, scores, image in zip(all_poses, all_scores, model_input):
+        for r, image in zip(result, model_input):
```

Comment:
Not important, you can skip, but I would write
- `result ` -> `batch_results` or `batch_predictions`
- `r` -> `sample_results` or `sample_predictions`
But not so important

Reply:
Fixed in 86b515fc7b9e1bfca1ac33d8f1c2af58ba30543d, thanks :)